nav2_system_tests/params/keepout_params.yaml

```diff
@@ -0,0 +1,7 @@
+costmap_filter_info_server:
+  ros__parameters:
+    filter_info_topic: "costmap_filter_info"
+    type: 0
+    mask_topic: "filter_mask"
+    base: 0.1
+    multiploer: 0.2
```

Comment:
"multiplier"

Reply:
Fixed in https://github.com/ros-planning/navigation2/pull/1882/commits/a8fb4ed6703c56e10def3393ffa6cd96d8fdd8fc#diff-9ddfdbab21a7f3906c07bd77b6858340R7